R/boundsPostprob.R

```diff
@@ -1,44 +1,35 @@
 #' Decision cutpoints for boundary (based on posterior probability)
 #'
 #' This function is used to identify the efficacy and futility
-#' boundaries based on posterior probabilities, i.e.:
-#' Efficacy boundary: find minimum x (xU) where Pr(P>p0|x,n,a,b) >= tU and
-#' Futility boundary: find maximum x (xL) where Pr(P>p1|x,n,a,b) <= tL
+#' boundaries based on the following rules:
```

Comment:
there are a lot of changes here for `boundsPostprob`, and I see there is still the PR https://github.com/Genentech/phase1b/pull/104 ongoing - it would be better to first get that done and then continue here with `plotDiff`